src/main/java/org/terasology/gooeyDefence/events/combat/DoSelectEnemies.java

```diff
@@ -17,22 +17,40 @@
 
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.event.Event;
+import org.terasology.gooeyDefence.towerBlocks.base.TowerTargeter;
 
 import java.util.Collection;
 import java.util.HashSet;
 import java.util.Set;
 
+/**
+ * Event sent to select the enemies that will be attacked.
+ * Sent against the Emitter blocks in the tower.
+ *
+ * @see TowerTargeter
+ */
 public class DoSelectEnemies implements Event {
```

Comment:
Rename DoSelectEvents to SelectEnemeiesEvent?

Reply:
(Renamed in a later branch to avoid conflicts)